specification/advisor/resource-manager/Microsoft.Advisor/stable/2022-09-01/feedback.md

```diff
@@ -0,0 +1,55 @@
+### Feedback #1 ###
```

Comment:
Is this `feedback.md` referenced any where? swagger don't need this file

Reply:
This file is not needed. I think it was accidentally added to the PR.